go.mod

```diff
@@ -2,10 +2,13 @@ module github.com/hashicorp/terraform-provider-boundary
 
 go 1.19
 
+// TODO REMOVE THIS AFTER THE 0.12.0 RELEASE
+replace github.com/hashicorp/boundary => ../boundary
+
 require (
-	github.com/hashicorp/boundary v0.11.1
-	github.com/hashicorp/boundary/api v0.0.32
-	github.com/hashicorp/boundary/sdk v0.0.26
+	github.com/hashicorp/boundary v0.12.0
+	github.com/hashicorp/boundary/api v0.0.33
+	github.com/hashicorp/boundary/sdk v0.0.29
```

Comment:
These should be updated to the latest, which I believe is `v0.0.34` for api and `.v0.0.30` for sdk.

Reply:
Thanks!